src/lib.ts

```diff
@@ -377,20 +379,27 @@ export function safeRestore(): void {
   burnLibrams(mpTarget * 2); // Leave a mp buffer when burning
 }
 
+/**
+ * Compares the local version of Garbo against the most recent release branch, printing results to the CLI
+ */
 export function checkGithubVersion(): void {
   if (process.env.GITHUB_REPOSITORY === "CustomBuild") {
     print("Skipping version check for custom build");
   } else {
     const gitBranches: { name: string; commit: { sha: string } }[] = JSON.parse(
       visitUrl(`https://api.github.com/repos/${process.env.GITHUB_REPOSITORY}/branches`)
     );
-    const mainBranch = gitBranches.find((branchInfo) => branchInfo.name === "main");
-    const mainSha = mainBranch && mainBranch.commit ? mainBranch.commit.sha : "CustomBuild";
-    if (process.env.GITHUB_SHA === mainSha) {
+    const releaseCommit = gitBranches.find((branchInfo) => branchInfo.name === "release")?.commit;
```

Comment:
Can we move this logic & the gitBranches logic into the `else` statement?

Reply:
Good call, yes, yes we can. One sec